src/Orckestra.Composer.Website/App_Data/Razor/Composer/Product/Summary.cshtml

```diff
@@ -25,7 +25,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://www.composite.net/ns/function/1.0">
 <head>
     <script type="text/javascript">
-        var productDetailContext = '@Model.JsonContext';
```

Comment:
tryeto use [HttpUtility.JavaScriptStringEncode](https://learn.microsoft.com/en-us/dotnet/api/system.web.httputility.javascriptstringencode)  if you want to encode charcters